tests/Listener/SubSelectPalettesListenerTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/**
+ * @package    meta-palettes
```

Comment:
Fix header plz.

Reply:
Done in 361e073a066b8c29eb9938d4dbc57520140a0efa.